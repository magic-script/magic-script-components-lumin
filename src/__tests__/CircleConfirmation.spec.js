//
import React from "react";
import { create } from "react-test-renderer";
import { CircleConfirmation } from "magic-script-components";

describe("CircleConfirmation component", () => {
  test("Matches the snapshot", () => {
    const circleconfirmation = create(<CircleConfirmation />);
    expect(circleconfirmation.toJSON()).toMatchSnapshot();
  });
});
