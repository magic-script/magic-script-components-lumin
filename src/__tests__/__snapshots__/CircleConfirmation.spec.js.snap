// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CircleConfirmation component Matches the snapshot 1`] = `<circleConfirmation />`;
